src/Compilers/CSharp/Portable/BoundTree/Constructors.cs

```diff
@@ -281,9 +281,10 @@ internal partial class BoundIndexerAccess
                 expanded: false,
                 argsToParamsOpt: default(ImmutableArray<int>),
                 defaultArguments: default(BitVector),
-                originalIndexers,
+                originalIndexersOpt: originalIndexers,
                 type: indexer.Type,
-                hasErrors: true);
+                hasErrors: true,
+                accessorKind: AccessorKind.Unknown);
```

Comment:
Minor: Consider adding `accessorKind: AccessorKind.Unknown` in the order matching the parameters (that is, following `expanded: false`) if the arguments are otherwise in parameter order.